Tasks/plane.js

```diff
@@ -1,18 +1,16 @@
 // Make nested array plane
-
-const_plane = function(arr, res=[]) {
-  j = 0; for(let i = 0, length = arr.length; i < length; i++){
-    value = arr[i];
-    j = i
-    if (Array.isArray(value, typeof value) && [i, length]) {
-    res.push(...const_plane(value));
-    arr[i] = res[i - 1]
-    }else{
-    arr[i] = res[j - 1]
-    res.push(value);
-    };
-  };
-  return res
-}
-
-require('../Tests/plane.js')(const_plane);
+'use strict';
+const planedArray = (arr) => {
+  const res = [];
+  for (const value of arr) {
+    if (Array.isArray(value)) {
+      res.push(...planedArray(value));
+    } else {
+      res.push(value);
+    }
```

Comment:
Alternatively you can use ternary here, but it's ok

Reply:
P.P.S I`ve changed 'else-if' block into the ternary operator as well. This advice was alternative, hence I`ve marked it as 'optional'